uint16 ID
string bbox_class
float64 probability
float64 xmin
float64 ymin
float64 xmax
float64 ymax
geometry_msgs/PoseStamped pose
yolov8_ros/SkeletonPoint[] skeleton
